### PR TITLE
[hellporno] Add extraction for .net zone, fix tests (wrong extension)

### PR DESCRIPTION
**Pull request by savicki**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x ] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x ] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x ] Bug fix
- [ x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

### CODE REVIEW

File: youtube_dl/extractor/hellporno.py [hunk 1]
```diff
@@ -38,7 +49,9 @@ def _real_extract(self, url):
 
         video_id = flashvars.get('video_id')
         thumbnail = flashvars.get('preview_url')
-        ext = flashvars.get('postfix', '.mp4')[1:]
```
Comment (savicki) [inline youtube_dl/extractor/hellporno.py]: valeu always "360p.mp4" which failed test_1, at least

File: youtube_dl/extractor/hellporno.py [hunk 2]
```diff
@@ -10,8 +10,8 @@
 
 
 class HellPornoIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?hellporno\.com/videos/(?P<id>[^/]+)'
-    _TEST = {
+    _VALID_URL = r'https?://(?:www\.)?hellporno\.((com/videos)|(net/v))/(?P<id>[^/]+)'
```
Comment (dstftw) [inline youtube_dl/extractor/hellporno.py]: Don't capture groups you don't use.

File: youtube_dl/extractor/hellporno.py (hunk 1 above)
Comment (dstftw) [inline youtube_dl/extractor/hellporno.py]: `utils.determine_ext`.

File: youtube_dl/extractor/hellporno.py [hunk 3]
```diff
@@ -22,7 +22,18 @@ class HellPornoIE(InfoExtractor):
...
+        'url' : 'http://hellporno.net/v/186486/',
+        'md5': '65adb331709d68d01067527d5547ae88',
+        'info_dict': {
+            'id': '186486',
+            'display_id': '186486',
+            'ext': 'mp4',
+            'title': 'Tattooed brunette tries anal sex with a big cock',
+            'thumbnail': 're:https?://.*\.jpg/$',
+            'age_limit': 18,
+        }
```
Comment (dstftw) [inline youtube_dl/extractor/hellporno.py]: Should be `only_matching`.

**Comment by savicki** [comment]
flake8 give 2 long lines warnings (e501)

**Pull request closed without merging**